services/orchestrator-agent/tests/unit/test_config_validation.py

```
@@ -22,7 +22,16 @@ def test_missing_db_password_rejected_in_production(self):
             validate_secrets("production", "", "real-token")
 
     def test_valid_secrets_pass_in_production(self):
-        assert validate_secrets("production", "s3cret", "token") == []
+        assert (
+            validate_secrets(
+                "production",
+                "s3cret",
+                "token",
+                gitops_webhook_token="gitops-token",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "gitops_webhook_token"

(S106)